docs2/site/docs/migrations/migration4.md

```diff
@@ -2,43 +2,87 @@
 
 ## New Features
 
-* Extension methods to configure authorization requirements for GraphQL elements: types, fields, schema.
-* New property `GraphQL.Introspection.ISchemaComparer ISchema.Comparer { get; set; }`
-* New property `IResolveFieldContext.ArrayPool`
+### Improved Performance
+
+GraphQL.NET 4.0 has been highly optimized, typically executing queries at least 50% faster while also providing a 65% memory reduction. Small queries have been measured to run as much as twice as fast as previously. A cached query executor is also provided, which can reduce execution time another 20% once the query has been parsed (disabled by default). Variable parsing is also improved to run about 50% faster, and schema build time is now about 20x faster than previously and requires 1/25th the amount of memory.
```

Comment:
```suggestion
GraphQL.NET 4.0 has been highly optimized, typically executing queries at least 50% faster while also providing a 65% memory reduction. Small queries have been measured to run as much as twice fast than previously. A cached query executor is also provided, which can reduce execution time another 20% once the query has been parsed (disabled by default). Variable parsing is also improved to run about 50% faster, and schema build time is now about 20x faster than previously and requires 1/25th the amount of memory.
```
?

Also please break long lines.

Reply:
"twice as fast as" is correct... but we could either make it shorter

"measured to run ~as much as~ twice as fast as previously"

and at the end, "previously" could instead be written as "the previous version" or "they previously ran".

"measured to run as much as twice as fast as ~previously~ the previous version."

"measured to run as much as twice as fast as ~previously~ they previously ran."

```suggestion
GraphQL.NET 4.0 has been highly optimized, typically executing queries at least 50% faster while also providing
a 65% memory reduction. Small queries have been measured to run twice as fast as they previously ran. A cached
query executor is also provided, which can reduce execution time another 20% once the query has been parsed
(disabled by default). Variable parsing is also improved to run about 50% faster, and schema build time is now
about 20x faster than previously and requires 1/25th the amount of memory.
```

Don't worry about the line breaks.  It's still a WIP and I'll get it all cleaned up.  I only just started on this.